### PR TITLE
infra: Use SWC builder

### PR DESCRIPTION
**Pull request by XPECTER**
# Pull Request Template

## Type of PR

- [x] Feature - Platform
- [ ] Feature - Game
- [ ] Fix - Bug

## Description of PR

Changed to use SWC Builder when building.
- time taken for build
  - tsc : 4.84s user 0.14s system 277% cpu 1.798 total
  - swc : 0.29s user 0.06s system 116% cpu 0.300 total
- time taken for test
  - tsc : 9.34s user 1.09s system 198% cpu 5.265 total
  - swc : 4.65s user 0.83s system 183% cpu 2.992 total

## Changes Made

Add `.swcrc`. This is configuration file.
Add dependency `@swc/core`, `@swc/cli`

## Testing Method

`time npm run test` or `time npm run compile`

## Checklist

- [x] Prepared for code review (Assignees set, Labels applied, etc.)
- [x] Checked that changes do not negatively affect existing features
- [ ] Documentation completed for new features
- [ ] All unit tests passed
- [ ] Testing in the test environment completed
- [ ] Linked this PR in the related issue
- [ ] Communicated sufficiently with the relevant team or person before submitting PR

### CODE REVIEW

File: package.json
Comment (junman95) [inline package.json]: I suppose this version will be downgraded to 1.80 due to supporting  compatibility.
This pr will get a conflict after merging #22 .

**Comment by junman95** [comment]
Nice one!
I will approve this pr after merging 0.2.0 pr #22

File: package.json
Comment (junman95) [reply package.json]: @XPECTER 
I fixed conflict on package.json directly.
Check the last commit i made on this PR.

**Review by Ooyd — APPROVED** [review]

**Review by smshack — APPROVED** [review]
lgtm

**Review by junman95 — APPROVED** [review]

**Pull request merged**
